### PR TITLE
Moving xcodeproj customized configs to xcconfig files.

### PR DESCRIPTION
**Pull request by rdeassis**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Moving all the configurations for all targets that were customized in the xcodeproj file into specific xcconfig files.

### Verification

Verified all settings remain unchanged comparing Xcode settings(customized tab) before and after they were moved to xcconfig files.

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui-apple/pull/320)

### CODE REVIEW

File: ios/xcode/FluentUITests.xcconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
+CODE_SIGN_STYLE = Automatic
+DEVELOPMENT_TEAM = UBF8T346G9
```
Comment (kubalani) [inline ios/xcode/FluentUITests.xcconfig:8]: Do we need this?

Comment (rdeassis) [reply ios/xcode/FluentUITests.xcconfig:8]: We don't necessarily need it to run the tests. But it'll avoid the warnings in Xcode:

Screen Shot 2020-11-19 at 8 48 03 PM

File: ios/xcode/FluentUILib.xcconfig [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+CODE_SIGN_STYLE = Automatic
+MTL_FAST_MATH = YES
+OTHER_LDFLAGS = -ObjC
+PRODUCT_NAME = FluentUI
```
Comment (Saadnajmi) [inline ios/xcode/FluentUILib.xcconfig:9]: +1 for not having this hardcoded in the pbxproj now :D

**Review by Saadnajmi — APPROVED** [review]

File: ios/xcode/FluentUITests.xcconfig (hunk 1 above)
Comment (harrieshin) [reply ios/xcode/FluentUITests.xcconfig:8]: i don't think i like developement team tag in public?

Comment (harrieshin) [reply ios/xcode/FluentUITests.xcconfig:8]: also that is specific to office team afaik

Comment (rdeassis) [reply ios/xcode/FluentUITests.xcconfig:8]: This was already public in the project file. These are just being moved to xcconfig files.

That is the Apple Team ID for Microsoft.
It's part of a lot of App Bundles distributed publicly by Microsoft.
An example is Teams App which has this in the embedded.provisionprofile file inside its application Bundle.

```
	<key>TeamIdentifier</key>
	<array>
		<string>UBF8T346G9</string>
	</array>
```

**Review by harrieshin — APPROVED** [review]

**Pull request merged**